extensions/blocks/business-hours/index.js

```diff
@@ -82,7 +83,7 @@ export const settings = {
 	title: __( 'Business Hours', 'jetpack' ),
 	description: __( 'Display opening hours for your business.', 'jetpack' ),
 	icon,
-	category: 'jetpack',
+	category: typeof registerBlockCollection === 'function' ? 'widgets' : 'jetpack',
```

Comment:
I would centralize `@wordpress/blocks` import and typeof check into a helper function at `extensions/shared/block-category.js` and do `supportsCollections()` or similar.